staging/src/k8s.io/kube-aggregator/pkg/apiserver/metrics.go

```diff
@@ -46,7 +50,36 @@ var x509InsecureSHA1Counter = metrics.NewCounter(
 	},
 )
 
+var (
+	extensionApiserverRequestCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "extension_apiserver_requests_total",
+			Help:           "Counter of extension apiserver request broken down by result. It can be either 'OK', 'Not Found', 'Service Unavailable' or 'Internal Server Error'.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"result"},
+	)
+
+	extensionApiserverLatency = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Name:           "extension_apiserver_request_duration_seconds",
+			Help:           "extension apiserver request duration in seconds broken out by result.",
+			Buckets:        metrics.ExponentialBuckets(0.001, 2, 15),
```

Comment:
what's the ratio behind these pretty coarse buckets?

Reply:
For reference, request_duration_seconds has `Buckets: []float64{0.005, 0.025, 0.05, 0.1, 0.2, 0.4, 0.6, 0.8, 1.0, 1.25, 1.5, 2, 3, 4, 5, 6, 8, 10, 15, 20, 30, 45, 60}`.

Reply:
Same here: the lack of apiservice name or something like that surprises.